.github/workflows/ci.yml

```diff
@@ -0,0 +1,26 @@
+name: ci
```

Comment:
Please give the workflow and the filename a better name e.g. `docker`

Reply:
renamed